cmake/OpenEXR.pc.in

```diff
@@ -9,6 +9,7 @@ libdir=@PKG_CONFIG_INSTALL_LIBDIR@
 includedir=@PKG_CONFIG_INSTALL_INCLUDEDIR@
 OpenEXR_includedir=${includedir}/OpenEXR
 libsuffix=@LIB_SUFFIX_DASH@
+version=@OPENEXR_VERSION@
```

Comment:
What's the need for this extra custom variable, I don't see it being used?

Reply:
It is indeed in support of the python wheel setuptools, which needs the version info for both OpenEXR and Imath. But I can make it work with --modversion, and I agree that less is better, thanks.